### PR TITLE
:sparkles: Support Docker

### PR DESCRIPTION
**Pull request by HidemaruOwO**
## What
Support docker.
And add docker compose example

## Issue
#3 

## Changed

- Conventional NCFS to be a resident script because in the past, the Docker container would terminate when writing records, causing the background Ngrok process to also end

- Enabled port forwarding for IP addresses other than localhost. (Use `DOCKER_MC_NETWORK` environment) (Ex. DOCKER_MC_NETWORK=192.168.1.60 DOCKER_MC_NETWORK=mc)

- Add dockerfile and docker-compose

To confirm the operation before merging, please replace the Dockerfile with the following dockerfile.

```dockerfile
FROM ngrok/ngrok:alpine

USER root
# Install dependecies
# RUN apk update
RUN apk add --no-cache jq curl bash git shadow coreutils

# Create USER
RUN adduser --shell $(which bash) --disabled-password app

# Permission
RUN mkdir /app
RUN chown -R app /app

# Change user
USER app
WORKDIR /app

# Setup
# RUN git clone https://github.com/barbarbar338/ncfs.git /app/ncfs
RUN git clone https://github.com/HidemaruOwO/ncfs.git /app/ncfs
RUN cd /app/ncfs && git checkout feature/docker
RUN mv /app/ncfs/ncfs.sh /app
RUN chmod 755 /app/ncfs.sh

EXPOSE 4040
ENTRYPOINT [ "/app/ncfs.sh" ]
```

## Screenshots

<details>
<summary>Preview</summary>

- docker compose up

image

- connection

image
image
image

</details>

### CODE REVIEW

**Comment by HidemaruOwO** [comment]
I made a Dockerfile that retrieves files from Git, but is it better to COPY the files from the host?

File: docker/README.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+## Docker
+
+### Use docker compose
+
+- Create config.json
+
+Download conifg.json
```
Comment (barbarbar338) [inline docker/README.md]: Can you fix this typo before merging? :)

Comment (HidemaruOwO) [reply docker/README.md]: got it

**Comment by barbarbar338** [comment]
Everything looks okay, thank you for your contribution! 🎊

**Pull request merged**